### PR TITLE
Fixed typo in resource pool usage.

### PR DESCRIPTION
**Pull request by fireboy1919**
### What is this PR for?
This pull request adds fixes a typo in the resource pool system.

### What type of PR is it?
Bug Fix

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-749

### How should this be tested?
As this is a refactor only fix, no testing is appropriate.

### Screenshots (if appropriate)
Same as above.

### Questions:
* Does the licenses files need update?
No
* Is there breaking changes for older versions?
No
* Does this needs documentation?
No.

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Thanks for the fix. LGTM

File: zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterServer.java [hunk 1]
```diff
@@ -47,6 +47,7 @@
 import org.slf4j.LoggerFactory;
 
 import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
```
Comment (prabhjyotsingh) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterServer.java]: remove this.

Comment (fireboy1919) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterServer.java]: Done.

**Comment by jongyoul** [comment]
LGTM

**Comment by Leemoonsoo** [comment]
Merge if there're no more discussions

**Pull request closed without merging**